### PR TITLE
created data script

### PR DESCRIPTION
**Pull request by tak074**

### CODE REVIEW

File: DB/index.js [hunk 1]
```diff
@@ -14,3 +14,27 @@ const db = mongoose.connection;
...
+
+const {Client} = require('pg');
+console.log('first');
+const {user} = require('./environment.js');
+const {password} = require('./environment.js');
+const {host} = require('./environment.js');
+const {port} = require('./environment.js');
+const {database} = require('./environment.js');
+console.log('second');
+
```
Comment (mpstanford) [inline DB/index.js:27]: good use of environment variables! maybe check out the node package "dotenv" at some point, if you haven't already

File: DB/seedDB.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+const faker = require('faker');
+const fs = require('fs');
+
+(function generateData(number, batch) {
+  console.time('timer');
+  const clothes = ['hoodies', 'shirts', 'dress', 'shoes', 'pants', 'jacket'];
+  const randClothesIndex = Math.floor(Math.random() * clothes.length);
+  const keyword = clothes[randClothesIndex];
+  const pictureLength = Math.ceil(Math.random() * 7);
```
Comment (mpstanford) [inline DB/seedDB.js:9]: cool that you added this bit of complexity with the randomly-generated picture length to challenge yourself

File: DB/seedDB.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        fs.appendFileSync('./DB/primaries.csv', data);
+        console.log('adding: ', i);
+      } catch (err) {
+        console.log('error', i);
+        throw err;
+      }
+      counter = 0;
+      data = '';
+    }
+
```
Comment (mpstanford) [inline DB/seedDB.js:50]: may be worth checking to see if removing "console.log('adding: ', i)" cuts down on your overall time

**Review by mpstanford — APPROVED** [review]
Looks great, TK!!!

File: DB/seedDB.js (hunk 3 above)
Comment (tak074) [reply DB/seedDB.js:50]: that's a really good point! I will try that out

**Pull request merged**